run_FFT_analyzer.py

```diff
@@ -32,9 +35,9 @@ def run_FFT_analyzer():
                     FFT_window_size_ms  = 60,    # Window size used for the FFT transform
                     updates_per_second  = 1000,  # How often to read the audio stream for new data
                     smoothing_length_ms = 50,    # Apply some temporal smoothing to reduce noisy features
-                    n_frequency_bins    = 400,   # The FFT features are grouped in bins
+                    n_frequency_bins = args.frequency_bins, # The FFT features are grouped in bins
                     visualize = 1,               # Visualize the FFT features with PyGame
-                    verbose   = 1,               # Print running statistics (latency, fps, ...)
+                    verbose   = args.verbose,    # Print running statistics (latency, fps, ...)
```

Comment:
Note that verbose is now false by default. I found that the massive amount of printing can actually slow things down (especially on less capable hardware).